File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        
	int n = (int)s.size();
	int i = n-1 , length = 0;

	while(s[i] == ' ')
		i --;

	while(i >= 0 && s[i] != ' '){
		length ++;
		i --;
	}

	return length;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        
        l = s.split(" ")
        for i in range(len(l) - 1, -1, -1):
            if len(l[i]) == 0:
                continue
            else:
                return len(l[i])
